src/macros.rs

```diff
@@ -421,3 +421,56 @@ macro_rules! rawdoc {
         object
     }};
 }
+
+/// Same as [`serde_with::serde_conv!`], but with support for rustdoc documentation for the struct.
+#[macro_export]
+macro_rules! my_serde_conv {
+    ($(#[$meta:meta])* $m:ident, $t:ty, $ser:expr, $de:expr) => {$crate::serde_conv!(pub(self) $m, $t, $ser, $de);};
```

Comment:
I don't think this matcher line is needed (and I think it'd be a compile error if anything actually matched on it, since there's no `$crate::serde_conv!` macro).